server/src/main/java/com/warehousepro/service/AuthenticationService.java

```diff
@@ -0,0 +1,33 @@
+package com.warehousepro.service;
+
+import com.warehousepro.dto.request.LoginRequest;
+import com.warehousepro.dto.response.ApiResponse;
+import com.warehousepro.dto.response.LoginResponse;
+import com.warehousepro.entity.User;
+import com.warehousepro.repository.UserRepository;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE , makeFinal = true)
+public class AuthenticationService {
+
+    UserRepository userRepository;
+
+    public LoginResponse authenticate(LoginRequest loginRequest){
```

Comment:
Change it to response User entity instead of dto, because you have to use this data to call again to generate jwt token

Reply:
```suggestion
    public User authenticate(LoginRequest loginRequest){
```